src/engine/renderer/glsl_source/reliefMapping_fp.glsl

```diff
@@ -43,7 +44,142 @@ vec3 NormalInTangentSpace(vec2 texNormal)
 	vec3 normal;
 
 #if defined(r_normalMapping)
-#if defined(USE_HEIGHTMAP_IN_NORMALMAP)
+#if defined(USE_NORMALMAP_FROM_HEIGHTMAP)
```

Comment:
This section is pretty long, would you mind putting it in a function?

Reply:
Right.

Reply:
done